Makefile

```diff
@@ -6,4 +6,4 @@ build:
 	cd cmd/flowhouse; go build
 
 bindata:
-	cd pkg/frontend; go-bindata -pkg frontend assets/
+	cd pkg/frontend; ~/go/bin/go-bindata -pkg frontend assets/
```

Comment:
Uhm, that works for you but not for me. You should install go-bindata into $PATH.

Reply:
sorry, fixed!